extensions/prisma/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zed_prisma"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
Let's remove the version bump and just do the fix in this PR.

We can bump the extension version afterwards.